plugins/module_utils/k8s/runner.py

```diff
@@ -57,6 +58,38 @@ def run_module(module) -> None:
         msg = "Failed to load resource definition: {0}".format(e)
         raise CoreException(msg) from e
 
+    select_all = module.params.get("select_all")
```

Comment:
This block can be isolated in a `get_definitions()` function.

Reply:
done